lib/v2/screens/profile_screens/guardians/guardians_tabs/guardians_screen.dart

```diff
@@ -13,7 +12,7 @@ class GuardiansScreen extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     return BlocProvider(
-        create: (context) => GuardiansBloc()..add(LoadGuardians(userName: SettingsNotifier.of(context).accountName)),
+        create: (context) => GuardiansBloc(),
         child: BlocListener<GuardiansBloc, GuardiansState>(
             listenWhen: (context, state) => state.pageCommand != null,
             listener: (context, state) {
```

Comment:
I think we must follow this rule.

https://dart.dev/guides/language/effective-dart/style#prefer-using-_-__-etc-for-unused-callback-parameters
from this: `listenWhen: (context, state) => state.pageCommand != null, `
to this: `listenWhen: (_, current) => current.pageCommand != null,`

Reply:
agree

Reply:
We can make lint detect these.